sdk/servicebus/service-bus/src/receivers/receiverCommon.ts

```diff
@@ -186,7 +218,10 @@ function settleMessage(
 
   let error: Error | undefined;
   if (message.delivery.remote_settled) {
-    error = new Error(MessageAlreadySettled);
+    error = translateServiceBusError({
+      description: MessageAlreadySettled,
+      condition: ErrorNameConditionMapper.SessionLockLostError
```

Comment:
Reason for using "SessionLockLostError" as the condition? This code is also used by non sessions right?

Reply:
Ah, fixed to conditionalize this based on the sessionId property.